src/database/models/index.js

```diff
@@ -25,11 +25,10 @@ fs.readdirSync(__dirname)
       file.indexOf('.') !== 0 && file !== basename && file.slice(-3) === '.js'
   )
   .forEach((file) => {
-    const model = require(path.join(__dirname, file)).default(
+    const model = require(path.join(__dirname, file))(
```

Comment:
Add back this `default` function. Otherwise, the models written in ES6 would throw an error then, of course, change all the models to be written in ES6.